src/new-client/src/components/tearOutContext.tsx

```diff
@@ -0,0 +1,15 @@
+import { createContext } from "react"
```

Comment:
components probably isn't the place for this context file, I would put it alongside the tear out state file

Reply:
I ended up putting it in` @/App/TearoutSection `because the state file for the tear out sections was within `@/Web`